doc/man3/DTLSv1_listen.pod

```diff
@@ -64,6 +64,23 @@ does not support this), then B<*peer> will be cleared and the family set to
 AF_UNSPEC. Typically user code is expected to "connect" the underlying socket to
 the peer and continue the handshake in a connected state.
 
+Warning: It is essential that the calling code connect the underlying socket to
+the peer after making use of DTLSv1_listen(). In the typical case where
+L<BIO_s_datagram(3)> is used, the peer address is updated when receiving a
+datagram on an unconnected socket. If the socket is not connected, it can
+receive datagrams from any host on the network, which will cause subsequent
+outgoing datagrams transmitted by DTLS to be transmitted to that host. In other
+words, failing to call BIO_connect() or a similar OS-specific function on a
+socket means that any host on the network can cause outgoing DTLS traffic to be
+redirected to it by sending a datagram to the socket in question. This does not
+break the cryptographic protections of DTLS but may facilitate a denial of
+service attack or allow unencrypted information in the DTLS handshake to be
```

Comment:
`denial of service attack` -> `denial-of-service attack` ?

Reply:
Hm, is there a reason to do this hyphenation?

Reply:
Well, I'm not a native speaker, but according to my understanding of English grammar,
when a composite noun is used as an adjective, it should be hyphenated, e.g.,  "state-of-the-art solutions" .
Just found this: https://www.eslbuzz.com/grammar-compound-adjectives-in-english/

Reply:
Fair enough.